src/main/java/net/authorize/api/controller/base/ApiOperationBase.java

```diff
@@ -252,7 +252,7 @@ private void validateAndSetMerchantAuthentication() {
 		}
 	}
 	
-	public synchronized String getClientId() {
+	public String getClientId() {
 		return Constants.CLIENT_ID;

```

Comment:
when was the synchronized added? Don't remember it being in the original implementation.,